prototypes/javaspec-engine/src/test/java/info/javaspec/engine/JavaSpecEngineTest.java

```diff
@@ -28,191 +29,264 @@
 	@Override
 	public void declareSpecs(JavaSpec javaspec) {
 		javaspec.it("can be loaded with ServiceLoader and located by ID", () -> {
-			EngineTestKit.engine("javaspec-engine").selectors(selectClass(nullSpecClass())).execute();
-		});
-
-		javaspec.it("#discover reports to a provided EngineDiscoveryRequestListener", () -> {
```

Comment:
White space differences are making this look like a bigger change than it is.  That's why I wanted to stop this PR after I added support for a single level of `describe` blocks.

The big win: now the specs for the engine itself are better organized, by the method they are testing.